### PR TITLE
proxyless POC

### PR DESCRIPTION
**Pull request by capaj**
this is just a POC. 

Before we can fix the CI we would have to merge GET method support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,3 +106,95 @@ const deployedPromptCompletion = await lt.prompts.invoke({
...
+let's suppose you have a prompt called `joke-teller` deployed on staging in langtail. You can get it's
+
+```ts
+import { LangtailPrompts } from "langtail"
+
+const lt = new LangtailPrompts({
+  apiKey: "<LANGTAIL_API_KEY>",
+})
+
+const playgroundState = await ltLocal.get({
```
Comment (petrbrzek) [inline README.md:123]: It should be `await lt.get({`

File: README.md [hunk 2]
```diff
@@ -106,3 +106,95 @@ const deployedPromptCompletion = await lt.prompts.invoke({
 Full API reference is in [API.md](API.md)
 
 We support the same [runtimes as OpenAI](https://github.com/openai/openai-node?tab=readme-ov-file#requirements).
+
+### proxyless usage
+
+You can avoid langtail API all together by constructing your prompt locally and calling your provider like openAI directly.
+
+let's suppose you have a prompt called `joke-teller` deployed on staging in langtail. You can get it's
```
Comment (petrbrzek) [inline README.md:114]: There is an unfinished sentence here.

File: README.md [hunk 3]
```diff
@@ -106,3 +106,95 @@ const deployedPromptCompletion = await lt.prompts.invoke({
...
+                },
+              ],
+              "tools": [],
+              "type": "chat",
+            },
+          }
+```
+
+```ts
+const openAiBody = ltLocal.build(playgroundState, {
```
Comment (petrbrzek) [inline README.md:163]: It should be `lt` instead of `ltLocal`.

File: src/LangtailPrompts.spec.ts [hunk 4]
```diff
@@ -42,100 +42,185 @@ describe(
...
+
+        expect(res.httpResponse.status).toBe(200)
+        // TODO when we have some API for logs we could assert on that too that the log record does not have any payload
+      })
     })
 
-    it("should not record", async () => {
-      //full deployed prompt path "do-not-delete-api-key-used-on-ci-iSNqij/ci-tests-project/short-story-teller/staging"
-      const ltWithProject = new LangtailPrompts({
+    describe.only("build", () => {
```
Comment (petrbrzek) [inline src/LangtailPrompts.spec.ts:154]: Do not forget to remove `.only`.

Comment (capaj) [reply src/LangtailPrompts.spec.ts:154]: Yes. I will before I move this out of draft.

**Comment by capaj** [comment]
superseeded by #13

**Pull request closed without merging**